Automattic/jetpack
# Search: Move customizer integration into package

Fixes #23153.

#### Changes proposed in this Pull Request:

- Moves Instant Search integration code for the WordPress Customizer into the Search package.
- Removes now obsolete references to the `modules/search` folder in the Jetpack plugin.





#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your site with a Search subscription.
* Navigate to the WordPress Customizer.
* Ensure that the Jetpack Search section still exists and remains usable.
